source/MaterialXCore/Document.cpp

```diff
@@ -601,6 +601,21 @@ void Document::upgradeVersion()
         minorVersion = 36;
     }
 
+    // Upgrade path for 1.37 (change types to child outputs)
+    for (NodeDefPtr nodeDef : getNodeDefs())
```

Comment:
This is the upgrade path for old documents which had NodeDefs with type attributes. Removes these attributes and creates a child output called "out".